docs/setup_linux.md

```diff
@@ -0,0 +1,74 @@
+## Setting up an agent on Linux
+
+On Linux, it is recommended to setup the Agent using the Docker file provided in this repository.
+There instructions are valid for x86 and arm.
```

Comment:
s/x86/x86_64/
s/arm/ARM64 (aka ARMv8 or AARCH64)/

Reply:
Where is this feedback syntax coming from? I get it, took me the second example though.